neuronpp/utils/record.py

```diff
@@ -1,11 +1,9 @@
-from nrn import Segment, Section
```

Comment:
I think this concept of recording with labels should be moved to separated class which inherits from `Record`, eg. `RecordWithLabels`. In this case - `dt` and `true_labels` params from the `_plot_animate()` metod can be moved into consructor.